pkgs/by-name/la/lan-mouse/package.nix

```diff
@@ -0,0 +1,50 @@
+{ rustPlatform
+, fetchFromGitHub
+, lib
+, pkg-config
+, glib
+, gtk4
+, libadwaita
+, xorg
+, libX11 ? xorg.libX11
+, libXtst ? xorg.libXtst
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "lan-mouse";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "feschber";
+    repo = "lan-mouse";
+    rev = "v${version}";
+    hash = "sha256-98n0Y9oL/ll90NKHJC/25wkav9K+eVqrO7PlrJMoGmY=";
+  };
+
+  nativeBuildInputs = [
+    glib # needed in both {b,nativeB}uildInptus
+    pkg-config
+  ];
+
+  buildInputs = [
+    glib
+    gtk4
+    libadwaita
+    libX11
+    libXtst
+  ];
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "reis-0.1.0" = "sha256-sRZqm6QdmgqfkTjEENV8erQd+0RL5z1+qjdmY18W3bA=";
+    };
+  };
+
+  meta = with lib; {
+    description = "Mouse & keyboard sharing via LAN";
+    homepage = "https://github.com/feschber/lan-mouse";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ pedrohlc ];
+    platforms = lib.platforms.unix;
```

Comment:
upstream claims windows support as well

Reply:
My bad, I thought that Cygwin/Mingw would fall under Unix as well.